plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/impl/menu/AbstractMenu.java

```diff
@@ -54,22 +55,27 @@ public void select(){
 	public boolean isSelected(){
 		return mh.isSelected(menuItem);
 	}
-	
+
 	@Override
 	public String getText() {
 		return mh.getMenuItemText(menuItem).replace("&", "");
 	}
-	
+
+	@Override
+	public String getLabel() {
+		return mh.getLabelFromText(getText());
```

Comment:
removed method getText() from ShellMenu and redundant code from MenuHandler.getLabelFromText()